client/devdocs/controller.js

```diff
@@ -139,8 +138,7 @@ const devdocs = {
 	},
 
 	pleaseLogIn: function ( context, next ) {
-		const currentUrl = new URL( window.location.href );
-		const redirectTo = currentUrl.protocol + '//' + currentUrl.host + '/devdocs/welcome';
```

Comment:
I was wondering if this removes a seam for testing, but it looks like there are ways to mock `window.location` if necessary, so 👍

Reply:
Both the old and new code use `window.location` -- the new one is just a bit simpler. So, there should be the same mocking opportunities as before.